addons/account/models/chart_template.py

```diff
@@ -269,7 +269,7 @@ def tax_template_changed(tax, template):
                         if (
                             command not in (Command.UPDATE, Command.CREATE)
                             or not self.ref(vals['tax_src_id'], raise_if_not_found=False)
```

Comment:
```suggestion
                            or not self.ref(vals.get('tax_src_id'), raise_if_not_found=False)
```

Reply:
The reason for not using get over here was that the `tax_src_id` is a required field. So, we can use get if you want ??

Reply:
For this one I'll let @odoo/rd-accounting decide